### PR TITLE
Update Developer Guide's Product Scope and User Stories

### LINKED ISSUE #19: As a developer, I can view the Product Scope in the Developer Guide
... so that I can understand the intended purpose of the product.

### PR DESCRIPTION
**Pull request by FredericChow00**
# What

Update Developer Guide to include product scope and user stories

# Why

Developer Guide was still from the default AB3 Project

# How

Included user stories brainstormed by the team as well as our value proposition when deciding on proceeding with this product 

Fix #19

### CODE REVIEW

File: docs/team/fredericchow00.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: <Frederic Chow> Project Portfolio Page
+---
+
+### Project: HMHero
+
+### What should be included
+
+- [x] Code contirbuted
```
Comment (Merrickneo) [inline docs/team/fredericchow00.md:10]: Typo here! Change contirbuted to contributed!

**Review by Merrickneo — CHANGES_REQUESTED** [review]
Small typo in your portfolio file.

File: docs/team/fredericchow00.md (hunk 1 above)
Comment (FredericChow00) [reply docs/team/fredericchow00.md]: Oops sorry, corrected! Thanks for pointing out

**Review by Merrickneo — APPROVED** [review]
Saw the edit. LGTM

**Pull request merged**